subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r40/ProjectConfigurationChildrenProgressCrossVersionSpec.groovy

```diff
@@ -274,23 +274,23 @@ class ProjectConfigurationChildrenProgressCrossVersionSpec extends ToolingApiSpe
 
         applyBuildScript.child("Resolve dependencies of :compileClasspath").with {
             it.child "Configure project :a"
-            it.descendant "Download http://localhost:${server.port}${projectB.pomPath}"
-            it.descendant "Download http://localhost:${server.port}/repo/group/projectC/maven-metadata.xml"
-            it.descendant "Download http://localhost:${server.port}${projectC.pomPath}"
-            it.descendant "Download http://localhost:${server.port}${projectD.metaDataPath}"
-            it.descendant "Download http://localhost:${server.port}${projectD.pomPath}"
+            it.descendant "Download ${server.uri}${projectB.pomPath}"
```

Comment:
why do we assume the uri ends with a delimiter?

Reply:
We don't. `${projectB.pomPath}` always stars with a `/` delimiter.